src/main/java/core/basesyntax/service/impl/ParserDataImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.service.ParserData;
+
+public class ParserDataImpl implements ParserData {
+    public static final int OPERATION_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUALITY_INDEX = 2;
+
+    @Override
+    public Transaction parseData(String data) {
```

Comment:
where are you actually using validator?
can't find it in this class and also in main()